README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
-    * Each block has a stand alone purpose, i.e. each function provides a specific functionality
-    * Data Scientists can create their own blocks, i.e. create own R functions
-* Constructing ADaM dataset should become like building out of blocks that are based on `{admiral}` modular functions and user created modular functions
+One of the key aspects of `{admiral}` is its development by the users for the users.
+It gives an entry point for all to collaborate, co-create and contribute to a
+harmonised approach of developing ADaMs in R across the pharmaceutical industry.
 
-## Expectations
+## Scope
 
-* `{admiral}` will never cover 100% of eventualities that could be needed to produce ADaMs across each and every company/disease area/study - ADaM is infinite
-* Some flexibility can be added to the functions, but only where there is an agreed common need across the industry as this has to balance vs ease of usage and testing
-* One of our principle design decisions in creating `{admiral}` was to prioritize transparency and simplicity for our users - and not to let this ever become a "black-box" toolkit
-* We hope `{admiral}` offers a chance for users to be programmers - this is not a "run 1 line and an ADaM appears" solution or an attempt to automate ADaM
-* It is expected for companies to adopt `{admiral}` that a company-specific extension package would likely be needed (e.g. `{admiralroche}` or `{admiralgsk}`)
-    * Consider this an opportunity to influence company-specific standards, as `{admiral}` offers a bridge towards a more industry-aligned implementation of ADaM
-* From the core `{admiral}` package covering common functions and mostly safety templates, there will be further package extensions dedicated to certain disease area endpoints (e.g. `{admiralonco}` or `{admiralhiv}`)
+To set expectations: It is not our target that `{admiral}` will ever provide all possible solutions
+for all ADaM datasets outside of study specific needs. It depends on the user's collaboration
+and contribution to help grow over time to an asset library that is robust, easy to use and
+has an across-industry focus. We do not see a coverage of 100% of all ADaM derivations as ever
+achievable - ADaM is endless.
+
+We will provide:
+* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a
+modular manner (an "opinionated" design strategy)
+* Pharmaceutical communities and companies are encouraged to contribute to `{admiral}` following
+the provided programming strategy and modular approach
+* Functions that are comprehensively documented and tested, including example calls - these are
+all listed in the [Reference section](https://pharmaverse.github.io/admiral/reference/index.html)
+* Vignettes on how to create ADSL, BDS and OCCUR datasets, including example scripts
+* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation,
```

Comment:
Is it possible to link these out  to the vignettes?  If folks come to the github and scroll down to the REadme it would be a good chance to send them to the part of the website with user friendly vignettes.  The reference section makes my eyes hut

Reply:
you mean to have them linked to the example scripts and vignettes directly from the readme?

Reply:
yes